client-course-schedulizer/src/utilities/helpers/teamMembers.ts

```diff
@@ -0,0 +1,49 @@
+export interface TeamMember {
+  bio: string;
+  name: string;
+  photo: string;
+  website: string;
+}
+
+/* Basic information about the project team members */
+export const team: TeamMember[] = [
+  {
+    bio:
+      "Computer Science student at Calvin University. Jonathan is a hard worker, a faster runner, and has previous web development experience.",
```

Comment:
Capitalizing student to remain more consistent with capitalizing professor and department chair (but if you disagree that's fine too)
```suggestion
      "Computer Science Student at Calvin University. Jonathan is a hard worker, a faster runner, and has previous web development experience.",
```